lib/globals.d.ts

```diff
@@ -25,7 +25,7 @@ declare namespace NodeJS {
     Command: any;
```

Comment:
I was thinking something at the top of the file to describe why the entire file exists (e.g. ambient typings that aren't captured in the rest of our type dependencies, but we can revisit this.